CIScripts/Build/BuildFunctions.ps1

```diff
@@ -161,7 +160,6 @@ function Invoke-ExtensionBuild {
         if($BuildMode -eq "debug") {
```

Comment:
could you please remove that check? we'd like to have pdbs also in production mode